gamedata/scripts/z_auto_looter.script

```diff
@@ -224,6 +224,16 @@ function cond_ammo_disassemble(obj)
 			return true
 		end
 	end
+	if config.disassemble_ammo == disassemble_ammo_options.junk_itms then
```

Comment:
This `if` is redundant. Function returns `false` by default, so if it doesn't hit the line 234, then it will be false anyway

Reply:
Removed this if function